docs/NEW_LANGUAGE_FEATURE_INTEGRATION_PROCESS.md

```diff
@@ -0,0 +1,31 @@
+# Check Update Procedure
+
+## 1. Check Discovery Methods
+
+### 1.1 Consider Similar Tokens
+### 1.2 Java Enhancement Proposals (JEPS)
+### 1.3 Always Impacted Checks
```

Comment:
Please add sub headers here, for whitespace, illegaltoken, illegaltokentext

Reply:
As bullet points ok? And what about indentation should we include it as always impacted

Reply:
We can do bullet points, this is a good idea

Reply:
Yes, let's count indentation in whitespace for sure

Reply:
done